### ISSUE TITLE
Question about key.priv and address.txt

### ISSUE THREAD

**Issue by whitenoiseL** [paritytech/parity-deploy#48]
Just two quick questions about key.priv and address.txt. 

Why the address I generate from private key which is stored in key.priv is different from the address that is stored in address.txt?

What if I want to use my own account as miner in aura?

**Comment by whitenoiseL** [comment]
I find the way to use my own account as miner finally. It turns out that even if I remove key.priv, key.pub and address.txt, the POA chain can still be built up without error. So I guess these three files have nothing to do with the chain configuration. Then what are they generated for?

**Comment by ddorgan** [comment]
I think there was some change here. Let me look into this and get back to you.

**Comment by whitenoiseL** [comment]
@ddorgan It seems there are something wrong with the generated files (i.e key.priv, key.pub), therefore the enodes in reserved_peers file are also wrong. 
If we start a POA chain, let's say a 2-node-POA, those two nodes are not connected with each other.  We need to correct the information that is stored in reserved_peers to make them connected. For me, I get the right enode information by post parity_enode method to my authority node. 
I guess there is something wrong in parity-deploy.sh file. I haven't look into that yet.

**Comment by ddorgan** [comment]
@whitenoiseL yes there were some changes in a recent release that I need to build into parity-deploy. Will keep you posted once I have a PR merged.

**Comment by ddorgan** [comment]
@whitenoiseL please checkout the lastest master and let me know if this helps. It's been switched to use parity-ethereum stable. 

Let me work on the key issues beyond that, as I'll need to run quite a lot of tests on different versions to make sure the update works everywhere.

**Comment by whitenoiseL** [comment]
@ddorgan The latest change doesn't help much. The issue is still the same. The address in address.txt does not match the address generated from private key which is stored in key.priv. 
The enode information changes everytime when I rebuild the chain. Does that mean the enode information can only be got after the chain is up? Or do I understand it wrong.

![wechat screenshot_20180817230256](https://user-images.githubusercontent.com/16938270/44273551-9c3aee00-a1f4-11e8-8f03-e34f08fb742e.png)

**Comment by ddorgan** [comment]
Can you please run sudo bash clean.sh and then run something like parity-deploy.sh --config aura --nodes 3 (for example).

**Comment by whitenoiseL** [comment]
@ddorgan Well, it didn't work. The enodes are still not match.
![wechat screenshot_20180818000114](https://user-images.githubusercontent.com/16938270/44276427-f5993080-a279-11e8-9079-3a277d795f8c.png)

**Comment by ddorgan** [comment]
Please ensure you have parity stable (v1.11.8) installed locally (in /usr/local/parity) and try again, thanks.

**Comment by whitenoiseL** [comment]
@ddorgan Yeah, thanks a lot! It works now! All nodes are connected. However the enodes show from parity still don't match what is stored in reserved_peers. But they do connected now. I am still trying to figure out why. Thanks anyway!
